grazer/src/main/webapp/resources/js/searchConcepts.js

```diff
@@ -0,0 +1,26 @@
+$(document).ready(function() {
+    $("#conceptQuery").autocomplete({
+        minLength: 1,
+        autofocus: true,
+        source: function (request, response) {
+            var pathArray = window.location.pathname.split( '/' );
+            var root = pathArray[1];
+
+            $.getJSON('/' + root + '/searchPage/getConcepts', request, function(result) {
```

Comment:
that whole path splitting is pretty ugly and likely to break when things are not standard (e.g. if the app is not directly deployed under root, but myserver.org./soemthing/grazer). Just wrap this in an init method that is called from the jsp and have the init method require the app root path. Then you can use jsp to get the app root and simply pass that on to this script.